lib/decoder.js

```diff
@@ -980,15 +984,18 @@ module.exports = decode;
 function decode(jpegData, opts) {
   var defaultOpts = {
     useTArray: false,
-    colorTransform: true
+    colorTransform: true,
+    tolerantDecoding: true
```

Comment:
to avoid any future breaking change based on tolerantDecoding, how do you feel about this being off by default?